diagnostic.md

```diff
@@ -3,67 +3,59 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+#ajax-error-message, .container, .js-stale-session-flash, #facebox
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+jquery("li") and $("li")
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
 
 ## Question 4
 
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+Getter
+$("h1").val()
+$("title").attr()
 
-<!-- your answer ends here -->
+Setter:
+$("h1").val("Help!")
+$("title").attr("Help!")
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM is a representation of the HTML using JavaScript objects. The DOM is created and loaded into the browser when the HTML is loaded and is subsequently made into Objects that can be manipulated using JavaScript. Each element is made into a Node which represents an object that can be manipulated using javascript methods. The manipulation allows for a more dynamic user experience.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.